### PR TITLE
[WT-187] feat/account-panel-tests

### PR DESCRIPTION
**Pull request by rafijv**
Add e2e tests
- Invite friend
- Change first, last name

### CODE REVIEW

File: test/e2e/cypress/integration/account-panel.spec.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+describe('Account panel', () => {
+  const firstName = 'Name';
+  const lastName = 'Lastname';
+  const email = 'name@intx.com';
```
Comment (fabioespinosa) [inline test/e2e/cypress/integration/account-panel.spec.ts]: its good to keep this practice of using @inxt.com theres a typo there though

File: src/app/shared/components/Input/index.tsx [hunk 2]
```diff
@@ -29,6 +30,7 @@ export default function Input({
   onBlur?: () => void;
   message?: string;
   autofocus?: boolean;
+  name?: string;
```
Comment (fabioespinosa) [inline src/app/shared/components/Input/index.tsx:33]: nice addition to the Input component

File: src/app/shared/components/Input/index.tsx [hunk 3]
```diff
@@ -95,6 +97,7 @@ export default function Input({
           setIsFocused(false);
         }}
         value={value}
+        name={name}
```
Comment (fabioespinosa) [inline src/app/shared/components/Input/index.tsx:100]: i'm just curious if `name` is undefined will it render name="undefined" in the html 🤔  or it won't render name at all. 

If it is the former then there is no problem with this approach.

File: test/e2e/cypress/integration/account-panel.spec.ts [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+      cy.login();
+      cy.visit('/preferences?tab=account');
+    });
+
+    it('Should edit name and lastname', () => {
+      cy.get('button').contains('Edit').click();
+      cy.get('input[name=firstName]').type(firstName);
+      cy.get('input[name=lastName]').type(lastName);
+      cy.get('button').contains('Save').click();
+      cy.contains(firstName);
```
Comment (fabioespinosa) [inline test/e2e/cypress/integration/account-panel.spec.ts]: Can you please check if this `contains` awaits the change? or does it check the already typed result.

File: test/e2e/cypress/integration/account-panel.spec.ts [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    });
+
+    it('Should invite friend', () => {
+      cy.get('input:first').should('have.attr', 'placeholder', 'Enter friend email').type(email);
+      cy.get('button').contains('Send invitation').click();
+      cy.get('button').contains('See all invitations').click();
+      cy.contains(email);
+    });
+
+});
```
Comment (fabioespinosa) [inline test/e2e/cypress/integration/account-panel.spec.ts]: Try to use prettier to always have a EOF line

**Review by fabioespinosa — CHANGES_REQUESTED** [review]

File: src/app/shared/components/Input/index.tsx (hunk 3 above)
Comment (rafijv) [reply src/app/shared/components/Input/index.tsx:100]: does not render name

File: test/e2e/cypress/integration/account-panel.spec.ts (hunk 4 above)
Comment (rafijv) [reply test/e2e/cypress/integration/account-panel.spec.ts]: Yes, wait for the change

**Review by fabioespinosa — APPROVED** [review]

**Pull request merged**